tensorflow_probability/python/stats/sample_stats_test.py

```diff
@@ -683,6 +683,129 @@ def test_windowed_mean_corner_cases(self):
         [0., 1., 1.5, 2.5], self.evaluate(tfp.stats.windowed_mean(y)))
 
 
+@test_util.test_all_tf_execution_regimes
+class WindowedStatsTest(test_util.TestCase):
+
+  def _maybe_expand_dims_to_make_broadcastable(self, x, shape, axis):
```

Comment:
These two functions are as complex as the thing we're testing. Is there any way we can write this via np.vectorize?

Reply:
I refactored using np.vectorize, but I am not sure it is easier to read.